coriolisclient/v1/replicas.py

```diff
@@ -79,3 +79,14 @@ def delete_disks(self, replica):
 
         return replica_executions.ReplicaExecution(
             self, response.json().get("execution"), loaded=True)
+
+    def update(self, replica, updated_values):
+        data = {
+            "replica": updated_values
+        }
+        response = self._put(
+            '/replicas/%s' % base.getid(replica), data)
+        response = response.to_dict()
+
+        return replica_executions.ReplicaExecution(
+            self, response.get("execution"), loaded=True)
```

Comment:
Why does the above `delete_disks` call need to call `.json()` on the `response` and this doesn't?

Reply:
Fixed, was using the incorrect method before